modules/bitgo/src/v2/coins/near.ts

```diff
@@ -269,13 +340,274 @@ export class Near extends BaseCoin {
       txHex: serializedTx,
     } as any;
   }
+  // TODO(BG-51092): Needs to be moved to a common place to re-use it for other coins
+  static async initializeMPCIfNotPresent() {
+    if (!this.initialized) {
+      const hdTree = await Ed25519BIP32.initialize();
+      this.MPC = await Eddsa.initialize(hdTree);
+      this.initialized = true;
+    }
+  }
+
+  // TODO(BG-51092): Needs to be moved to a common place to re-use it for other coins
+  async getTSSSignature(
+    userSigningMaterial: UserSigningMaterial,
+    backupSigningMaterial: BackupSigningMaterial,
+    path = 'm/0',
+    transaction: accountLib.Near.Transaction
+  ) {
```

Comment:
lets also specify the function return type also

Reply:
Done